src/components/List/ListItem.tsx

```diff
@@ -32,18 +32,23 @@ export interface ListItemProps extends UIComponentProps, ContentComponentProps<a
 
   /** A list item can indicate that it can be selected. */
   selection?: boolean
```

Comment:
where and how this prop is used? it is very confusing to have both `selection` and `selected` prop for the same component

Reply:
this props defines if list is "selectable", what also includes navigable and focusable

Reply:
do you have any ideas on other naming? maybe just ```selectable``` can bring more meaning for that instead of ```selection```?